### PR TITLE
Update readme with prerequisite section and additional FAQ.

### LINKED ISSUE #42: Add notice on host parameter to readme
The host parameter should be the very first node ip address when using the Exasol subconnections. Providing a public domain name or public ip (for example, in AWS environment) address will not work.

More info on subconnections: [SOL-546](https://www.exasol.com/support/browse/SOL-546).

### PR DESCRIPTION
**Pull request by morazow**
- Adds a prerequisite section in order to use the connector.
- Adds description why `Connection refused` error might happen due to closed sub-connection ports.  

Fixes #42.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,21 @@ tables.
...
+* [Prerequisite](#prerequisite)
 * [Quick Start](#quick-start)
 * [Usage](#usage)
 * [Configuration](#configuration)
 * [Building and Testing](#building-and-testing)
 
+## Prerequisite
+
+* Deployed and running Spark cluster
+* Deployed and running Exasol cluster
```
Comment (3cham) [inline README.md]: I think a hint here for the number of executor condition would be great :)

Comment (morazow) [reply README.md]: I added short description. Please have a look if it makes sense.

On the FAQ, there is more detail about it, if Spark cannot start more tasks than Exasol nodes.

**Comment by 3cham** [comment]
LGTM :)

**Comment by morazow** [comment]
Thanks for the review @3cham! 
Could you please also approve from github review button? :) I changed the settings so that each pr requires at least one review before the merge. (but seems like I am able still to merge due to admin rights)

**Pull request merged**